test/support/page_objects/visualize_page.js

```diff
@@ -372,14 +369,7 @@ export default class VisualizePage {
   // this is for starting on the
```

Comment:
Is this comment relevant? Opening a saved visualization doesn't seem to have anything to do with the create new visualization page.